### DISCUSSION TITLE
Adding GraXpert as a process icon?

### DISCUSSION THREAD

**Discussion by chicago925** [DeepSkyForge/GraXpert4PixInsight#3] [Q&A]
If I add the GraXpert script as a process icon to my saved process icons and then load it, there are multiple errors in the process console:

*** PixInsight API Error: MetaProcess: GraXpert: Process parameter allocation failure: correction
*** PixInsight API Error: MetaProcess: GraXpert: Process parameter allocation failure: backgroundExtractionAIModel
*** PixInsight API Error: MetaProcess: GraXpert: Process parameter allocation failure: denoiseAIModel

Is there a way to avoid this? I don't see this with other scripts I have added as process icons.

**Comment by DeepSkyForge** [comment]
Yes, I opened case #8 and I'll provide a fix.
Sorry for that, I never tested this scenario before.
My bad.
Regards

**Comment by chicago925** [comment]
Thanks - much appreciated!

**Reply by DeepSkyForge** [reply]
Case #8 fixed in version 2.0.2 just released.
Have nice week-end :-)

**Comment by DeepSkyForge** [comment]
Just in case you didn't yet get it, the fix has been delivered 5 days ago.
I close the topic ;-)
